test/extended/util/oauthserver/oauthserver.go

```diff
@@ -97,7 +99,9 @@ func DeployOAuthServer(oc *exutil.CLI, idps []osinv1.IdentityProvider, configMap
 	if _, err := oc.AdminKubeClient().RbacV1().ClusterRoleBindings().Create(oauthSARolebinding); err != nil {
 		return nil, cleanups, err
 	}
-
+	if err := exutil.WaitForUserBeAuthorized(oc, "system:serviceaccount:"+oc.Namespace()+":e2e-oauth", "get", "oauthclients.oauth.openshift.io"); err != nil {
```

Comment:
Wait for the route `/healthz` to report `200` as well.

Reply:
Done in https://github.com/openshift/origin/pull/23528